eclair-core/src/main/scala/fr/acinq/eclair/payment/handlers/ReceiveHandler.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 ACINQ SAS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.acinq.eclair.payment.handlers
+
+import akka.actor.ActorContext
+import akka.event.LoggingAdapter
+
+
+trait ReceiveHandler {
+
+  def handle(implicit ctx: ActorContext, log: LoggingAdapter): PartialFunction[Any, Unit]
```

Comment:
Why not use `Actor.Receive` instead of `PartialFunction[Any, Unit]` ? It would be more explicit

Reply:
Addressed with 0f4d028.